src/main/java/wolox/training/model/Book.java

```diff
@@ -0,0 +1,55 @@
+package wolox.training.model;
```

Comment:
I think this is from a commit of pr #2 . For correction (not for merge) this pr should be into branch `feat-book` and not into `main`.
After merging #2 into `main`, this pr should return to point to `main`.